packages/hardhat-ethers/test/no-account.test.ts

```diff
@@ -0,0 +1,129 @@
+import { assert } from "chai";
```

Comment:
Do we have files with `.test.`?

Reply:
Not that I can see. It's currently unnecessary.

Reply:
I'd remove it, to keep things consistent.